### PR TITLE
feat(app): Spike theme variable organization

### LINKED ISSUE #27: Spike theme variable organization
Look into organization of theme variables. Real-life projects would have a more extensive set of colors, font-sizes, etc. What would the equivalent of a Sass _variables.scss file look like with styled-components?

### PR DESCRIPTION
**Pull request by kalongthewires**
* Convert `styles` folder to `themes`
* Add example of how theme variables could be organized and work
* Allow switching between themes via dropdown

Closes #27 

Putting this here for discussion. What do you two think?

### CODE REVIEW

**Comment by chriswhawkins** [comment]
I think I'm super excited to look at this!!!!

File: src/App.js [hunk 1]
```diff
@@ -1,105 +1,67 @@
...
+    this.setState({
+      theme: themes[target.value],
+    });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <label htmlFor="changeTheme">Change Theme:</label>
+        <select id="changeTheme" onChange={this.handleChange}>
```
Comment (csteinert) [inline src/App.js]: This is very fancy well done yes.

Comment (kalongthewires) [inline src/App.js]: Yay! 😂

**Comment by csteinert** [comment]
I'm curious about having zindex in the theme files. Those seem, to me, more like a base utility thing than something that's going to change on the whim of a theme. 🤔

**Comment by kalongthewires** [comment]
Per Caitlin's brilliant idea: going to create a z-index mixin/function as part of a separate issue. Removing z-index from this PR.

**Pull request merged**